elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -206,6 +206,7 @@ bool ElevationMapping::readParameters()
 bool ElevationMapping::initialize()
 {
   ROS_INFO("Elevation mapping node initializing ... ");
+  initialized_ = false;
```

Comment:
I think this variable should be set in the constructor, otherwise it will have a random value. I think adding -Wall and -Werror would give you an error in this case. (I would recommend adding them in another PR).